apps/web/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "./node_modules/@latitude-data/eslint-config/library.js",
+    "plugin:@next/next/recommended"
+  ],
+  "env": {
+    "node": true,
+    "browser": true
+  }
+}
```

Comment:
next is the only package i've not migrated to eslint 8 because of course it has some crazy dependencies with eslint 8 and migration is not straightforward.